commands.go

```diff
@@ -28,8 +28,9 @@ const (
 	deleteUserMessageType = messaging.MessageType(0x32)
 
 	// Shard messages
-	createShardGroupIfNotExistsMessageType = messaging.MessageType(0x40)
-	deleteShardGroupMessageType            = messaging.MessageType(0x41)
+	createShardGroupIfNotExistsMessageType    = messaging.MessageType(0x40)
```

Comment:
I see what you're saying about timestamps. But if the code computed the timestamp for the "precreate" message before it sent it out on the cluster -- it can, since it knows the shard -- and send the timestamp along, then the mesage signatures would be identical and you wouldn't need two messages.

Basically, I'm trying to get this implementation as simple as possible, without losing functionality. A "create shard group" message could be just that -- create these shards with these timestamps. Why the message is on the cluster, well, it wouldn't care.